### PR TITLE
Medic system update

### PR DESCRIPTION
**Pull request by GermanHydrogen**
Changes regarding the medic system, which were discussed last konvent.

- [x] Removed Ironpackplus
- [x] Added option to disable ace fatal damage for hardcore
- [x] Tweak KAT settings
  - Pneumothorax detection
  - Removal of autoinjector restrictions

### CODE REVIEW

File: addons/main/modKonfigurationsTemplates/medicSettings/hardcore.h [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+    force force ace_medical_fatalDamageSource = 1;
+#endif
+
 force force ace_medical_limping = 2;							// Man kann einstellen, ob man durch Wunden humpelt und wenn ja, ob Bandagieren das Humpeln stoppt
-force force ace_medical_fractures = 1; 							// Gibt es Knochenbr�che? (aka braucht man Schienen und wie viel bringen sie?)
-force force ace_medical_treatment_medicEpinephrine = 1; 		// Wer darf Epi benutzen?
+force force ace_medical_fractures = 1; 							// Gibt es Knochenbr�che? (aka braucht man Schienen und wie viel bringen sie?)
+force force ace_medical_treatment_medicEpinephrine = 0; 		// Wer darf Epi benutzen?
 force force ace_medical_treatment_medicPAK = 1; 				// Wer darf PAKs benutzen?
-force force ace_medical_treatment_medicSurgicalKit = 1; 		// Wer darf das Surgical Kit (N�hzeug) benutzen?
```
Comment (TAINCER) [inline addons/main/modKonfigurationsTemplates/medicSettings/hardcore.h:7]: Why is the encoding touched? Either fix the encoding or don't touch it.

**Review by TAINCER — CHANGES_REQUESTED** [review]
Please write out the words in the commit messages

File: addons/main/modKonfigurationsTemplates/medicSettings/hardcore.h [hunk 2]
```diff
@@ -1,15 +1,22 @@
 /* Hardcore */
-force force ace_medical_fatalDamageSource = 1; 					// Gibt es sofort t�dliche treffer? (Treffer in lebenswichtige Organe und den Kopf)
+
+// Gibt es sofort tödliche treffer? (Treffer in lebenswichtige Organe und den Kopf)
```
Comment (TAINCER) [inline addons/main/modKonfigurationsTemplates/medicSettings/hardcore.h:3]: Uppercase "Treffer"

**Review by TAINCER — CHANGES_REQUESTED** [review]

File: addons/main/modKonfigurationsTemplates/medicSettings/hardcore.h (hunk 1 above)
Comment (GermanHydrogen) [reply addons/main/modKonfigurationsTemplates/medicSettings/hardcore.h:7]: Wouldn't how to fix that

**Comment by GermanHydrogen** [comment]
> Please write out the words in the commit messages

Done

**Review by TAINCER — APPROVED** [review]
LGTM

**Pull request merged**